kumascript/macros/Deprecated_Inline.ejs

```diff
@@ -17,10 +17,8 @@
 var str = $0;
 var type = 0;   // assume we want the text badge
 
-if ($0 && $0 != undefined) {
```

Comment:
No need to check for `$0 != undefined`, since `$0` being truthy implies `$0 != undefined`.